File: php/tests/DiggingEstimatorTest.php
<?php

use kata\DiggingEstimator;
use PHPUnit\Framework\TestCase;

class DiggingEstimatorTest extends TestCase
{

    public function testShouldWork() : void {
        $estimator = new DiggingEstimator();
        $result = $estimator->tunnel(28, 2, "Granite"); // Works if you manually put [0, 3, 5.5, 7] for the rates

        $this->assertEquals(48, $result->total);
    }

}

File: java/src/test/java/DiggingEstimatorTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class DiggingEstimatorTest {

    @Test
    void tunnel() {
        var estimator = new DiggingEstimator(); // Works if you replace call to the service by [0, 3, 5.5, 7]
        var result = estimator.tunnel(28, 2, "GRANITE");
        assertEquals(48, result.total);
    }
}